app/models/start-task-info.ts

```diff
@@ -0,0 +1,22 @@
+import { Model, Prop, Record } from "app/core";
+import { TaskState } from "./task";
+
+export interface StartTaskInfoAttributes {
+    state: TaskState;
+    startTime?: Date;
+    endTime?: Date;
+    exitCode: number;
+    retryCount: number;
+}
+
+/**
+ * Class for displaying Start Task execution info from a node.
+ */
+@Model()
+export class StartTaskInfo extends Record<StartTaskInfoAttributes> {
+    @Prop() public state: TaskState;
+    @Prop() public startTime?: Date;
+    @Prop() public endTime?: Date;
```

Comment:
don't make some attribute here optional `?` only in the interface for the attributes